code/modules/events/pirates.dm

```diff
@@ -458,4 +452,13 @@
 
 /datum/export/pirate/cash/get_amount(obj/O)
 	var/obj/item/stack/spacecash/C = O
-	return ..() * C.amount * C.value
\ No newline at end of file
+	return ..() * C.amount * C.value
+	
+/datum/export/pirate/holochip
+	cost = 1
+	unit_name = "holochip"
```

Comment:
once again, subtype, you could avoid this entirely

Reply:
tbh i could just not do it since you can load chips directly, but one more export is hardly a problem